SpringProject/src/test/java/com/gradproject2019/AuthUtilsTest.java

```diff
@@ -0,0 +1,59 @@
+package com.gradproject2019;
+
+import com.gradproject2019.users.service.UserServiceImpl;
+import com.gradproject2019.utils.AuthUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AuthUtilsTest {
+
+    @Test
+    public void shouldCheckPasswordIsValidAgainstRegex() {
```

Comment:
Is an empty password valid? (applies to all fields)

Reply:
No because of the length assertion